SPECS/glibc/glibc.spec

```diff
@@ -27,14 +21,16 @@ Patch3:         CVE-2020-1751.nopatch
 # Rationale: Exploit requires crafted pattern in regex compiler meant only for trusted content
 Patch4:         CVE-2018-20796.nopatch
 Patch5:         glibc-2.34_pthread_cond_wait.patch
-
+%global security_hardening nonow
+%define glibc_target_cpu %{_build}
+%define debug_package %{nil}
+# Don't depend on bash by default
+%define __requires_exclude ^/(bin|usr/bin).*$
```

Comment:
Could you move these macros back to the top of the spec? That's where we usually try to keep them, so they are easier to find. The linter tends to move these around, unfortunately.